File: playwright_tests/lib/logout.py
from playwright.sync_api import Playwright, sync_playwright, expect, Page


def logout(page: Page) -> None:
    page.locator('[data-test="profile-icon"]').click()
    page.locator('[data-test="logout"]').click()
    expect(page.locator('[data-test="profile-icon"]')).not_to_be_visible()

File: playwright_tests/js/tests/lib/logout.ts
import { Page, expect } from '@playwright/test';


export const logout = async (page: Page) => {
    await page.goto("http://localhost:8000/");
    await page.locator('[data-test="profile-icon"]').click()
    await page.locator('[data-test="logout"]').click()
    await expect(page.locator('[data-test="profile-icon"]')).not.toBeVisible()
}